sourcemash/frontend/templates/base.html

```diff
@@ -30,18 +30,17 @@
 
   <body>
 
-    <div class="container">
-      {% block layout %}{% endblock %}
-    </div>
-
     <!--Import jQuery before materialize.js-->
-    <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
     {% assets "js_vendor" %}
       <script src="{{ ASSET_URL }}"></script>
     {% endassets %}
     {% assets "js_default" %}
       <script src="{{ ASSET_URL }}"></script>
     {% endassets %}
+    
+    <div class="container">
+      {% block layout %}{% endblock %}
+    </div>
```

Comment:
Because the feed_js assets imported require js_vendor and js_default, but we shouldn't load the assets in base.html because only feeds.html need them.